libs/csm/rest/csm_rest_bucket.py

```diff
@@ -26,6 +26,8 @@
 from commons.exceptions import CTException
 from commons.utils import config_utils
 from libs.csm.rest.csm_rest_test_lib import RestTestLib
+# pylint: disable-msg=unexpected-keyword-arg
+
```

Comment:
check for blank lines, if not needed we can remove

Reply:
Added a blank line as initially there was no separation between class and imports.